actioncable/lib/action_cable/subscription_adapter/redis.rb

```diff
@@ -150,6 +150,9 @@ def ensure_listener_running
 
                 conn = @adapter.redis_connection_for_subscriptions
                 listen conn
+              rescue ::Redis::BaseConnectionError
```

Comment:
Are there other IO/network/socket-related exceptions this should rescue?

Reply:
I don't believe so. This [error class](https://github.com/redis/redis-rb/blob/13c7a8e4fdb23d0fde8534ea974fdf089c8d11bc/lib/redis/errors.rb#L23-L25) covers all Redis related network exceptions.